test/find_test.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// find_test.dart
+// Tests for Find
+//
+// 2023 June 8
+// Author: Max Korbel <max.korbel@intel.com>
+//
+
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/rohd_hcl.dart';
+import 'package:rohd_hcl/src/utils.dart';
+import 'package:test/test.dart';
+
+void main() {
```

Comment:
we should test some more edge cases:
- what if N is 0 (specified, not null)?
- what if the first one found is the 0th bit?
- what if the first one found is the last bit?
- what if all 0's?
- what if all 1's?
- what if nothing is found? do we need another output for "not found"?

Reply:
done